fhir-server/src/main/java/com/ibm/fhir/server/resources/Capabilities.java

```diff
@@ -678,11 +653,12 @@ private CapabilityStatement buildCapabilityStatement(FHIRVersion fhirVersion) th
     }
 
     /**
+     * TODO: replace this with the new ResourcesConfigAdapter
```

Comment:
Will this be tackled in a future update?

Reply:
I guess there's no time like the present...

Reply:
removed this method.  required some other minor changes but nothing too bad